devimint/src/util.rs

```diff
@@ -596,6 +596,14 @@ impl Gatewayd {
             &[GATEWAYD_FALLBACK],
         ))
     }
+
+    /// Returns the gatewayd version from clap or default min version
+    pub async fn version_or_default() -> Version {
+        match cmd!(Gatewayd, "--version").out_string().await {
```

Comment:
I don't think we have `gatewayd --version` currently....

Reply:
`version-hash`? 

`handle_version_hash_command(fedimint_build_code_version_env!());`

Reply:
Ahh yes you're correct. Rather than changing `version_or_default()`, would it make sense to add a `--version` command to gatewayd?

Reply:
Yes we should add that

Reply:
@m1sterc001guy Good catch!

We're not currently parsing gatewayd versions for back-compat, but to be safe I added parsing to map version hash to version for the old binaries that won't have `--version`.

https://github.com/fedimint/fedimint/pull/4701